feeds/wlan-ap/wlan-ap-config/files/etc/uci-defaults/99-tip-data

```diff
@@ -2,22 +2,85 @@
 
 . /lib/functions.sh
 
+SKU="unknown"
+MODEL="unknown"
+PLATFORM="unknown"
+SERIAL="unknown"
+MODEL_REV="unknown"
+MODEL_DESCR="unknown"
+MANUF_NAME="unknown"
+MANUF_DATE="unknown"
+MANUF_URL="unknown"
+MANUF_DESIGN="unknown"
+REF_DESIGN="unknown"
+CERT_REGION="unknown"
+ID=""
+
 case "$(board_name)" in
 edgecore,ecw5211|\
 edgecore,ecw5410)
 	MODEL=$(cat /tmp/sysinfo/board_name | sed "s/edgecore,//" | tr [a-z] [A-Z])
-	SERIAL=$(cat /sys/class/net/eth0/address | tr -d :)
 	PLATFORM=$(cat /tmp/sysinfo/model)
+	SERIAL=$(cat /dev/mtd5 | grep serial_number | cut -d "=" -f2)
+	SKU=$(cat /dev/mtd5 | grep sku | cut -d "=" -f2)
+	CERT_REGION=$(cat /dev/mtd5 | grep certification_region | cut -d "=" -f2)
+	ID=$(cat /dev/mtd5 | grep mac_address | cut -d "=" -f2)
+	MANUF_DATE=$(cat /dev/mtd5 | grep manufacturer_date | cut -d "=" -f2)
+	MANUF_NAME="Edge-Core"
+	MANUF_ABBR="EC"
+	MANUF_URL=$(cat /dev/mtd5 | grep manufacturer_url | cut -d "=" -f2)
+	MODEL_DESCR=$(cat /dev/mtd5 | grep model_description | cut -d "=" -f2)
+	MODEL_REV=$(cat /dev/mtd5 | grep model_revision | cut -d "=" -f2)
+	REF_DESIGN=$(cat /dev/mtd5 | grep reference_design | cut -d "=" -f2)
 	;;
-cig,wf194c)
+cig,wf194c|\
+cig,wf188n)
 	MODEL=$(cat /tmp/sysinfo/board_name)
-	SERIAL=$(cat cat /dev/mtd14 | grep BaseMacAddress | cut -dx -f2 | tr -d '\r\n')
 	PLATFORM=$(cat /tmp/sysinfo/model)
+	SERIAL=$(cat /dev/mtd12 | grep serial_number | cut -d "=" -f2)
+	SKU=$(cat /dev/mtd12 | grep sku | cut -d "=" -f2)
+	CERT_REGION=$(cat /dev/mtd12 | grep certification_region | cut -d "=" -f2)
+	ID=$(cat /dev/mtd12 | grep mac_address | cut -d "=" -f2)
+	MANUF_DATE=$(cat /dev/mtd12 | grep manufacturer_date | cut -d "=" -f2)
+	MANUF_URL=$(cat /dev/mtd12 | grep manufacturer_url | cut -d "=" -f2)
+	MODEL_DESCR=$(cat /dev/mtd12 | grep model_description | cut -d "=" -f2)
+	MODEL_REV=$(cat /dev/mtd12 | grep model_revision | cut -d "=" -f2)
+	REF_DESIGN=$(cat /dev/mtd12 | grep reference_design | cut -d "=" -f2)
+	MANUF_NAME="CIG"
+	MANUF_ABBR="CIG"
 	;;
 linksys,ea8300)
 	MODEL=$(grep modelNumber= /dev/mtd9 | tr -d '\r\n' | sed "s/modelNumber=//")
 	SERIAL=$(grep serial_number= /dev/mtd9 | tr -d '\r\n' | sed "s/serial_number=//")
-	PLATFORM=OPENWRT_EA8300
+	PLATFORM="OPENWRT_EA8300"
```

Comment:
I am not sure we should change anything for Linksys (unless we maybe stub in values).  They are not a partner in the project and we have no expectation they will provide this data.

Actually I see the labels are different.  Does this mean they are providing this data in their default design with own labels?

Reply:
Mike was testing his changes against EA8300, hence I had to make these changes on the AP side. If the AP do not populate this data, then the AWLAN_Node will result in "unknown" for all the newly added parameters.

Yes, they seemed to provide most of the data except for "reference_design" and "sku_number". Only these fields go as "unknown" in the final output of AWLAN_Node.

Reply:
OK.